flagsmith-engine/identities/models.ts

```diff
@@ -29,6 +31,7 @@ export class IdentityModel {
         this.environmentApiKey = environmentApiKey;
         this.identifier = identifier;
         this.djangoID = djangoID;
+        this.transient = transient;
```

Comment:
These changes are not required. These models are used exclusively for local evaluation, for which we don't need transiency info.

Reply:
Removed